queue/src/fiskaltrust.Middleware.Localization.QueueDE/SignProcessorDE.cs

```diff
@@ -38,7 +38,7 @@ public class SignProcessorDE : IMarketSpecificSignProcessor
         public async Task<(ReceiptResponse receiptResponse, List<ftActionJournal> actionJournals)> ProcessAsync(ReceiptRequest request, ftQueue queue, ftQueueItem queueItem)
         {
             _logger.LogTrace("SignProcessorDE.ProcessAsync called.");
-            if (string.IsNullOrEmpty(request.cbReceiptReference) && !request.IsFailTransactionReceipt() && !string.IsNullOrEmpty(request.ftReceiptCaseData) && !request.ftReceiptCaseData.Contains("CurrentStartedTransactionNumbers"))
+            if (request.IsFailTransactionReceipt() && (string.IsNullOrEmpty(request.cbReceiptReference) || !request.ftReceiptCaseData.Contains("CurrentStartedTransactionNumbers")))
```

Comment:
@forsthug @volllly does someone remember why this check is here, and not in the `FailTransactionReceiptCommand`?

Reply:
I think that's just tech debt... AFAICT we can move that check into the `FailTransactionReceiptCommand` 👌

Reply:
@paulcristiann can you probably move the check to the mentioned receiptcommand?

Reply:
Yup, just moved it. However I think that we should also verify that the transactions exists when multiple should be closed, just as we do now in the case of a single transaction.